test/core/test_configs.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+from app.core.configs import DetectorConfig
+
+
+def test_detector_config():
```

Comment:
Awesome to have this test, maybe it would be good to name it something more specific like "test_detector_config_with_both_edge_modes" in case we want to add more detector config tests in the future?